composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Razorpay Magento 2.0 plugin for accepting payments.",
     "require": {
         "php": "~5.5.0|~5.6.0|^7.0",
-        "magento/framework": "~100.0||~101.0",
+        "magento/framework": "~102.0.0",    //Magento 2.3 compatible
```

Comment:
@seherkdc please remove the comments, it's an invalid JSON

Reply:
i did that still i received below errorDo not run Composer as root/super user! See https://getcomposer.org/root for details
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package magento/framework could not be found in any version, there may be a typo in the package name.

Reply:
@ccrims0n The comment has been removed. Kindly go ahead with the testing.